### PR TITLE
--certbot-dir is easier to set that --config-dir, --logs-dir, --work-dir

### PR DESCRIPTION
**Pull request by pde**
Implements #4638

### CODE REVIEW

File: certbot/cli.py [hunk 1]
```diff
@@ -1190,6 +1203,9 @@ def _paths_parser(helpful):
         help=config_help("work_dir"))
     add("paths", "--logs-dir", default=flag_default("logs_dir"),
         help="Logs directory.")
+    add("paths", "--certbot-dir", default=None,
+        help="Asks Certbot to keep all of its files below a given directory. --certbot-dir $DIR "
+             "is equivalent to --config-dir $DIR --work-dir $DIR/.internal/ --logs-dir $DIR/logs")
```
Comment (zjs) [inline certbot/cli.py]: Can you reference `constants.WORK_SUBDIR` and `constants.LOGS_SUBDIR` in the help message to ensure the message is updated if the constants ever change?

File: certbot/lock.py [hunk 2]
```diff
@@ -80,7 +80,9 @@ def _try_lock(self, fd):
                 logger.debug(
                     "A lock on %s is held by another process.", self._path)
                 raise errors.LockError(
-                    "Another instance of Certbot is already running.")
+                    "Another instance of Certbot is already running. If for "
+                    "some reason you really want to run two Certbots simultaneously, "
+                    "set --certbot-dir to a different value.")
```
Comment (zjs) [inline certbot/lock.py]: This seems distinct enough from adding `--certbot-dir` that it probably belongs in a separate PR.

It also seems like @bmw's comment from #4655, https://github.com/certbot/certbot/pull/4655#pullrequestreview-37966820, still applies:

> Additionally, I think this is the wrong place for this error message. If we fail to acquire a lock for the Apache or Nginx plugins, this exception about `--config-dir` and friends gets raised. Instead, if you'd like to see an error message like this, I think it should be added to `certbot.util.set_up_core_dir` by having us catch the `LockError` and raise a new error message. We already do this there in the event of a permission problem so adding another case for a `LockError` fits nicely in my opinion.

**Comment by bmw** [comment]
This has had unaddressed review feedback on it for a month and a half and when this was discussed in #3079, Jacob and I both agreed we liked a `--nosudo` or `--user` flag better than a flag like this as it's more friendly and less verbose to the average user. With this in mind, I'm closing this PR in favor of seeing one like we discussed in #3079 and/or review feedback being addressed.
